### PR TITLE
Add ForkLift.app v3.0 Beta 1

### PR DESCRIPTION
**Pull request by ArtemMedvedev**
### Checklist

- [x] The commit message includes the cask’s name and version.
- [x] `brew cask audit --download forklift-beta` is error-free.
- [x] `brew cask style --fix forklift-beta` left no offenses.

Additionally, when **adding a new cask**:

- [x] Checked there are no open [pull requests](https://github.com/caskroom/homebrew-versions/pulls) for the same cask.
- [x] Checked there are no closed [issues](https://github.com/caskroom/homebrew-versions/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install forklift-beta` worked successfully.
- [x] `brew cask uninstall forklift-beta` worked successfully.

### CODE REVIEW

File: Casks/forklift-beta.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+cask 'forklift-beta' do
+  version '3b1'
```
Comment (vitorgalvao) [inline Casks/forklift-beta.rb]: Make it instead `3.b1`.

File: Casks/forklift-beta.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  sha256 'abbad60a17efc262809e94517dfda90b510867942aa9e6de4a032c68bdd897c5'
+
+  url "http://download.binarynights.com/ForkLift#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/forklift-beta.rb]: Then here you use `#{version.no_dots}`.

File: Casks/forklift-beta.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  appcast 'http://update.binarynights.com/ForkLift3/update.xml',
```
Comment (vitorgalvao) [inline Casks/forklift-beta.rb]: And here and in `zap` you use `#{version.major}`.

**Comment by ArtemMedvedev** [comment]
@vitorgalvao done

**Pull request merged**